liquibase/scripts/drop_openmrs_schema.sql

```diff
@@ -0,0 +1,14 @@
+--
+--    This Source Code Form is subject to the terms of the Mozilla Public License,
+--    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+--    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+--    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+--
+--    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+--    graphic logo is a trademark of OpenMRS Inc.
+--
+use mysql;
+drop database openmrs;
```

Comment:
Will this work when the database name is not openmrs?

Reply:
No. Is there a scenario where the openmrs database would have a different name?

Reply:
Yes. I have heard of cases where two instances are installed on the same server but different database names.

Reply:
I see. These files are utility files used to generate new snapshot files. This happens from a fresh OpenMRS installation and not from an existing instance. Hence I suppose it is ok to use the standard database name.